### PR TITLE
Feat/register UI add radio buttons

### PR DESCRIPTION
**Pull request by emmanuelgaravito**
Added radio buttons as requested with validation and translation.

### CODE REVIEW

File: app/[locale]/(auth)/register/RegisterViewModel.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const RegisterViewModel = () => {
     defaultValues: {
       name: "",
       username: "",
+      radioButtons: undefined,
```
Comment (ingcarlostello) [inline app/[locale]/(auth)/register/RegisterViewModel.tsx]: change `radioButtons `to `usertype`

File: lib/validation.ts [hunk 2]
```diff
@@ -26,6 +26,9 @@ export const RegisterValidationSchema = (t: (arg: string) => string) =>
         .max(50, {
           message: t("USER_NAME_MAX_LENGTH"),
         }),
+      radioButtons: z.enum(["client", "business"], {
```
Comment (ingcarlostello) [inline lib/validation.ts]: change `business` to `businessOwner`

File: messages/en.json [hunk 3]
```diff
@@ -14,6 +14,9 @@
     "NAME": "Name",
     "USER_NAME": "User name",
     "NAME_TO_RECOGNIZE": "Name you will be recognized by",
+    "TYPE_OF_USER": "Type of user",
+    "CLIENT": "Client",
+    "BUSINESS": "Business",
```
Comment (ingcarlostello) [inline messages/en.json]: change `BUSINESS` to `BUSINESS_OWNER`

File: messages/en.json [hunk 4]
```diff
@@ -14,6 +14,9 @@
     "NAME": "Name",
     "USER_NAME": "User name",
     "NAME_TO_RECOGNIZE": "Name you will be recognized by",
+    "TYPE_OF_USER": "Type of user",
```
Comment (ingcarlostello) [inline messages/en.json]: change  `"TYPE_OF_USER": "Type of user"`, to `"USER_TYPE": "User Type"`

File: messages/es.json [hunk 5]
```diff
@@ -14,6 +14,9 @@
     "NAME": "Nombre",
     "USER_NAME": "Nombre de usuario",
     "NAME_TO_RECOGNIZE": "Nombre con el que te reconocerán",
+    "TYPE_OF_USER": "Tipo de usuario",
```
Comment (ingcarlostello) [inline messages/es.json]: apply the same as en.json file

File: messages/es.json [hunk 6]
```diff
@@ -14,6 +14,9 @@
...
+    "CLIENT": "Cliente",
+    "BUSINESS": "Negocio",
```
Comment (ingcarlostello) [inline messages/es.json]: change BUSINESS to BUSINESS_OWNER and spanish too

**Review by ingcarlostello — CHANGES_REQUESTED** [review]

File: app/[locale]/(auth)/register/RegisterViewModel.tsx (hunk 1 above)
Comment (emmanuelgaravito) [reply app/[locale]/(auth)/register/RegisterViewModel.tsx]: Done

File: lib/validation.ts (hunk 2 above)
Comment (emmanuelgaravito) [reply lib/validation.ts]: Done

File: messages/en.json (hunk 3 above)
Comment (emmanuelgaravito) [reply messages/en.json]: Done

File: messages/en.json (hunk 4 above)
Comment (emmanuelgaravito) [reply messages/en.json]: Done

File: messages/es.json (hunk 5 above)
Comment (emmanuelgaravito) [reply messages/es.json]: Done

File: messages/es.json (hunk 6 above)
Comment (emmanuelgaravito) [reply messages/es.json]: Done

**Review by ingcarlostello — APPROVED** [review]

**Pull request merged**